public/partial/recipe-detail-main.html

```diff
@@ -57,24 +57,24 @@
     <ng-include src="'partial/recipe-detail-indicators.html'"></ng-include>
     <ng-include src="'partial/recipe-detail-operations.html'"></ng-include>
     <div class="panel panel-default">
-        <div class="panel-heading"><strong>Nazi Mode</strong></div>
+        <div class="panel-heading"><strong>Modo estricto</strong></div>
         <div class="panel-body" >
             <div>
                 Habilitado
-                <span ng-show="recipe.naziMode" class="glyphicon glyphicon-check"/>
-                <span ng-hide="recipe.naziMode" class="glyphicon glyphicon-unchecked"/>
+                <span ng-show="recipe.strictMode" class="glyphicon glyphicon-check"/>
```

Comment:
If you modify this, you have to also modify backend i would't recomend to do this.

Reply:
yes, I can only change the FE... but as a plus I was removing "nazi" references all over the place...